release-please-config.json

```diff
@@ -21,16 +20,16 @@
       "bump-minor-pre-major": true
     },
     "packages/sdk-codegen-scripts": {
-      "release-as": "21.0.11"
+      "release-as": "21.0.12"
     },
     "packages/sdk-codegen-utils": {
-      "release-as": "21.0.11"
+      "release-as": "21.0.12"
     },
     "packages/sdk-codegen": {
-      "release-as": "21.0.11"
+      "release-as": "21.0.12"
     },
     "packages/sdk-rtl": {
-      "release-as": "21.0.11"
+      "release-as": "21.0.12"
```

Comment:
we may come up with another solution before the next release (e.g. renaming these packages) but for now this is their next version